.github/workflows/qit-self-group-test.yml

```
@@ -13,7 +13,7 @@ on:
 
 jobs:
   self_test_custom_test:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Thank you for the PR @Abdalsalaam! Can you please make one minor change? If we can use `ubuntu-24.04` instead of `latest` that would be preferable as this will match the logic on the QIT back end.

```suggestion
    runs-on: ubuntu-24.04
```

Thank you!

---

@zhongruige Thank you for reviewing this!